buildscripts/incremental/setup_conda_environment.sh

```diff
@@ -8,7 +8,11 @@ conda config --write-default
 conda config --set remote_connect_timeout_secs 30.15
 conda config --set remote_max_retries 10
 conda config --set remote_read_timeout_secs 120.2
-conda config --set restore_free_channel true
+if [[ $(uname) == Linux ]]; then
+    if [[ "$CONDA_SUBDIR" != "linux-32" || "$BITS32" != "yes" ]] ; then
```

Comment:
I think this needs to be an `&&` rather than an `||`.

Reply:
Thanks, I managed to invert only half the logic :)